### PR TITLE
Only use relative filepaths to session files

### PR DESCRIPTION
**Pull request by lauramurgatroyd**
Requires this PR: https://github.com/vais-ral/CILViewer/pull/133

- Ensures that the full path to the session folder is never saved. The app always works in the session folder, and uses relative paths to the files in the session folder. The only case where a full file path would be saved is if the setting to copy the image files to the session folder has not been selected (so the file lives outside of the session folder).

- This means a session folder could be moved to another computer, and the app would still be able to load it. (not possible in current version)

- In the case where a session has been loaded but an image file external to the folder has been deleted / cannot be loaded for some reason, an error dialogue appears which allows the user to load an image from another location. This allows the session to still be loaded rather than it crashing as it does currently.

### CODE REVIEW

File: src/idvc/dvc_interface.py [hunk 1]
```diff
@@ -513,27 +506,25 @@ def SelectImage(self, image_var, image, label=None, next_button=None):
...
-                    
+                        new_file_dest = os.path.join(file_name)
+
                     copy_worker = Worker(self.copy_file, start_location=f, end_location=new_file_dest)
                     self.threadpool.start(copy_worker)
                     files[file_num] = new_file_dest
-                    file_num+=1
-                    count+=1
+                    print("Current dir: ", os.getcwd())
+                    print("New file dest: ", new_file_dest)
```
Comment (paskino) [inline src/idvc/dvc_interface.py:523]: remove the prints

File: src/idvc/dvc_interface.py [hunk 2]
```diff
@@ -627,9 +619,10 @@ def view_image(self):
                 target_size = 0.125
         self.target_image_size = target_size
         
+        print("The in file is: ", self.image[0])
+        print("The current working directory is: ", os.getcwd())
```
Comment (paskino) [inline src/idvc/dvc_interface.py:623]: remove prints

**Comment by lauramurgatroyd** [comment]
This closes: https://github.com/TomographicImaging/iDVC/issues/13

**Comment by paskino** [comment]
I tried the app and get the following error

```
(dvc) PS C:\Users\ofn77899> idvc
QWindowsWindow::setGeometry: Unable to set geometry 2125x1987+0+0 (frame: 2125x1987+0+0) on QWidgetWindow/"VisualisationWidgetClassWindow" on "\\.\DISPLAY1". Resulting geometry: 2125x1987+1691+960 (frame: 2125x1987+1691+960) margins: 0, 0, 0, 0 minimum size: 54x54 MINMAXINFO maxSize=0,0 maxpos=0,0 mintrack=54,54 maxtrack=0,0)
[['C:/Users/ofn77899/Data/dvc/frame_000_f.npy'], []]
[['C:/Users/ofn77899/Data/dvc/frame_000_f.npy'], ['C:/Users/ofn77899/Data/dvc/frame_010_f.npy']]
The in file is:  ['C:/Users/ofn77899/Data/dvc/frame_000_f.npy']
The current working directory is:  C:\Users\ofn77899\DVC_Sessions\tmp_ai2jbla
Started worker
Spacing  (1.3088298169756785, 1.3088298169756785, 1.0)
Length of header:  96
array shape 2407406400
target 1073741824
Traceback (most recent call last):
  File "C:\Apps\miniconda3\envs\dvc\lib\site-packages\ccpi\viewer\utils\io.py", line 125, in <lambda>
    lambda: finish_fn(*finish_fn_args, **finish_fn_kwargs))
TypeError: save_image_info() got an unexpected keyword argument 'output_dir'
```

which I guess is due to https://github.com/vais-ral/CILViewer/pull/133

**Pull request merged**